src/components/Article.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => (
+  <>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </>
+);
+
+Article.defaultProps = {
+  date: new Date().toString(),
+  text: 'Some text',
+};
```

Comment:
Not the best solution. Better to make all fields required.

Article should be consistent object. And you definitely shouldn't render 'Some text' on the page.